components/pages/privacy/access/requests/index.jsx

```diff
@@ -39,35 +43,36 @@ import {
   TOS_PREDICATE,
 } from "../../../../../__testUtils/mockApp";
 import { getRequestorWebId } from "../../../../../src/models/consent/request";
-import { getProfileResource } from "../../../../../src/solidClientHelpers/resource";
 import Spinner from "../../../../spinner";
 
 const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
 export default function ConsentShow() {
   const { session } = useSession();
   const { fetch } = session;
-  const router = useRouter();
-  const { requestVc } = router.query;
+  const redirectUrl = window.location.href;
```

Comment:
I probably wouldn't store this as a constant, just do it inline in the `useEffect`, only `session.fetch` may change, but even then, that's unlikely